team-builder/src/components/Users.js

```diff
@@ -4,16 +4,16 @@ const Users = props => {
   console.log(props)
   return(
     <div>
-      {/* {props.addUserCB.map(user => {
+      {props.userList.map((user, i) => {
```

Comment:
added index to map

Reply:
oh, and we changed the addUserCB to what we were actually needing... userList. there is no prop addUserCB passed to this component...